packages/mongodb-artifact-generator/src/ArtifactGeneratorEnvVars.ts

```diff
@@ -2,4 +2,7 @@ import { CORE_ENV_VARS } from "mongodb-rag-core";
 
 export const ArtifactGeneratorEnvVars = {
   ...CORE_ENV_VARS,
+  GITHUB_ACCESS_TOKEN: "",
+  JIRA_USERNAME: "",
+  JIRA_PASSWORD: "",
```

Comment:
maybe don't put in core env vars in case someone wants to do the driver stuff but not this.

instead could make another const for this action and only assert that these are present when running the `generateReleaseNotes` action?